transcom/mymove
# B-17661 Add MTO Dates for TOO/TIO

Agility Ticket Number E-04636

## Summary

As a TOO and TIO I need to see the Scheduled and Actual Pickup Dates and the Scheduled and Actual Delivery

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access the officelocal
2. Login as a TIO or TOO
3. Check on move that has a MTO (not PPM).
4. Verify dates or emdash display in the table.

### Frontend


## Screenshots
